engine/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -14,6 +14,7 @@ storage driver as `overlay` or `overlay2`.
 > **Note**: If you use OverlayFS, use the `overlay2` driver rather than the
 > `overlay` driver, because it is more efficient in terms of inode utilization.
 > To use the new driver, you need version 4.0 or higher of the Linux kernel.
+> Or RHEL/Centos 7.4 or higher.
```

Comment:
This should be a comma, not a complete new sentence. What version of the kernel does 7.4 ship with? Is it lower than 4.0? If not, this extra clause seems unnecessary.

Reply:
Rhel/Centos 7.4 ships with the correct kernel that supports overlay2 of 3.10.0-693.

Reply:
The comma makes sense. The Rhel compatible kernel is 3.10.0-693.